src/Models/Material.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SolutionsService.Models
+{
+    public class Material
+    {
+        [Key]
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public virtual HowTo Solution { get; set; }
```

Comment:
Does this relationship have to be bidirectional? Are we ever requesting the solution from a material?

Reply:
For Entity Framework this is necessary for proper mapping